src/libopensc/padding.c

```diff
@@ -559,6 +568,7 @@ static int sc_pkcs1_add_pss_padding(sc_context_t *scctx, unsigned int hash, unsi
 	rv = SC_SUCCESS;
 
 done:
+	sc_log_openssl(scctx);
```

Comment:
I think this should be before the `goto done` as this code path is used also for the success.